### ISSUE TITLE
Can not import ngtpy with torch 1.10.0

### ISSUE THREAD

**Issue by shirayu** [yahoojapan/NGT#103]
This may be related to #37.

# Situation

The following code causes an error: ``free(): invalid pointer``.

```python
import ngtpy
import torch
```

torch 1.9.0 and 1.8.0 also cause the same error.
The following code causes no error.

```python
import ngt
import torch
```

# Expected behavior
can import properly

# Environment

```console
$  python3 -V
Python 3.9.7

$  cat /etc/debian_version
10.11

$  grep -A1 name poetry.lock
name = "ngt"
version = "1.12.2"
--
name = "numpy"
version = "1.21.1"
--
name = "pybind11"
version = "2.8.1"
--
name = "torch"
version = "1.10.0"
--
name = "typing-extensions"
version = "4.0.0"
```

- [gdb.txt](https://github.com/yahoojapan/NGT/files/7615051/gdb.txt)

**Comment by masajiro** [comment]
I have released [v1.14.3](https://pypi.org/project/ngt/1.14.3/#files) that fixed this issue.

**Comment by shirayu** [comment]
Thank you for the update.
The import works fine.

However, I got another error. So, I opened the issue #116.
